tasks/oah-update.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: Pulling latest fossology image
+  become: yes
+  community.docker.docker_image:
+    name: fossology/fossology
+    source: pull
+    state: latest
```

Comment:
Error - Value must be present or Absent (https://docs.ansible.com/ansible/latest/collections/community/docker/docker_image_module.html#parameter-state)
![image](https://user-images.githubusercontent.com/36243506/190999979-011ccfce-865e-4993-9f35-a8c7e1206f2f.png)